test/indent.txt

```diff
@@ -19,12 +19,74 @@ to go
 ++]
 end
 
+IfMultiBracket1
+to foo
+  let abc 0
+  if true [ set abc [
+    1 2 3 ]
+  ] [
+++  set abc [4 5 6]
+  ]
+end
+
+# I'm not certain this should be normative
+# This behavior wasn't specified prior to 6.0.1 and could probably change
+IfMultiBracket2
```

Comment:
Anyway, in response to your comment, this is a weird enough case that being normative about it seems odd. That said, it looks right to me. I don't know. Could go either way.